sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -524,24 +541,14 @@ public virtual async Task<Response<TableItem>> CreateIfNotExistsAsync(Cancellati
         /// Deletes the table specified by the tableName parameter used to construct this client instance.
         /// </summary>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>If the table exists, a <see cref="Response"/>. If the table already does not exist, <c>null</c>.</returns>
+        /// <returns>The <see cref="Response"/> indicating the result of the operation.</returns>
```

Comment:
Is a behavioral change in the return value?

Reply:
Yes, but intentional. The way we return null today can cause a very strange NRE. https://github.com/Azure/azure-sdk-for-net/issues/25434